scripts/k8s-deployment-sim.sh

```diff
@@ -66,17 +66,16 @@ kubectl describe services $(kubectl get services|grep agent-plg|awk '{print $1}'
 printf "\n\n\n*** [3.4] clear installation\n"
 kubectl delete --all deployments && kubectl delete --all pods && kubectl delete --all services
 
-printf "\n\n\n*** [4] install lber in k8s\n"
-#helm install k8s/agent+lber --generate-name
-: 'printf "\n\n\n*** [4.1] verify installation\n"
+printf "\n\n\n*** [4] install oauth in k8s\n"
+helm install k8s/examples/oauth --set-file global.oauthConfig=k8s/examples/oauth/oauth.env --generate-name
+printf "\n\n\n*** [4.1] verify installation\n"
 kubectl get deployments && kubectl get pods && kubectl get services && kubectl get ingresses
 printf "\n\n\n*** [4.2] verify deployment spec\n"
-kubectl describe deployments $(kubectl get deployments|grep agent-plg|awk '{print $1}'|head -n 1)
-printf "\n\n\n*** [4.2] verify statefulset spec\n"
-kubectl describe sts $(kubectl get sts|grep agent-plg|awk '{print $1}'|head -n 1)
+kubectl describe deployments $(kubectl get deployments|grep oauth|awk '{print $1}'|head -n 1)
 printf "\n\n\n*** [4.3] verify service spec\n"
-kubectl describe services $(kubectl get services|grep agent-plg|awk '{print $1}'|head -n 1)
+kubectl describe services $(kubectl get services|grep oauth|awk '{print $1}'|head -n 1)
```

Comment:
please describe pvc in minikube @ramaraosrikakulapu

Reply:
pvc tests are added now